File: mario.c

int main(void)
{
    int n;

    do {
        n = get_int("Height: ");
       }
    while (n < 1 || n > 8);      
                             // n = height

    if (n > 0 || n < 9)
      {
        int i = 0; 
                             // i = counter
                             
        for (int j = 0; j < n; j++) 
                                      // j = row
         {
            if (i != n) 
                                      // != - not equal
            {
                for (int k = (n - 1) - i; k > 0; k--) 
                                                         // k = spaces
                 {
                    printf(" ");
                 }
                for (int m = 0; m <= i; m++) 
                                                          // m = hashes                 
                 {
                    printf("#");
                 }

                printf("  ");

                for (int m = 0; m <= i; m++) 
                 {
                    printf("#");
                 }

                printf("\n");
                
                i++;
            }
          }     
        }       
}

File: mario.py
from cs50 import get_int
while True:
    # - height, m - width 
    n = get_int("Height: ")
    m = n + 1
    if n >= 1 and n <= 8:
        break
for i in range(0, n, ): 
    hashes = i + 1
    spaces = m - hashes

    print(" " * (spaces - 1), end="")
    print("#" * hashes)